example/gtfs-rt-simple.jv

```diff
@@ -0,0 +1,106 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+pipeline GtfsRTSimplePipeline {
+	block GTFSRTTripUpdateFeedExtractor oftype HttpExtractor {
+		url: "https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-trip-update";
+	}
+
+    block GTFSRTVehiclePositionFeedExtractor oftype HttpExtractor {
+		url: "https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-vehicle-position";
+	}
+
+    block GTFSRTAlertFeedExtractor oftype HttpExtractor {
+		url: "https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-alerts";
+	}
+
+    block GtfsRTTripUpdateInterpreter oftype GtfsRTInterpreter{
```

Comment:
Style: There should be a space before `{` (everywhere).

Reply:
changed